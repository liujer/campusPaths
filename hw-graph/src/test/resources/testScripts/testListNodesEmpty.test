# Tests listNodes method on an empty graph
CreateGraph B
ListNodes B