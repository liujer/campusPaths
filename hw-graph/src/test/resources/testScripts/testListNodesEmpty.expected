# Tests listNodes method on an empty graph
Created graph B
ListNodes of B output:
